src/test/java/org/takes/facets/auth/social/PsGithubTest.java

```diff
@@ -135,6 +152,16 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * A private method to create the basic directives, without access token.
```

Comment:
@sebing no need to mention `A private method to...`. `Creates basic directives without an access token.` is fine.